core/ingestion/upload/upload/upload.py

```diff
@@ -181,7 +183,10 @@ def upload_segment(params, meta, segment, blob, f):
     container = meta['guid']
 
 
-    blob.create_container(name=container)
+    try:
```

Comment:
Creating the container inside the upload_segment function was a bug. Although this patch fixes the issue it will still print the error message. create_container should be called before the segments are uploaded.

Reply:
In the tests we now delete all the containers for each run. So this is not needed anyway

Reply:
overwrite=True is not, but crate_container should be moved since it will cause issues when uploading multiple segments